cli/cmd/agent_aws-install_ec2ic.go

```diff
@@ -63,6 +63,11 @@ AWS credentials are read from the following environment variables:
 - AWS_SESSION_TOKEN (optional)
 - AWS_REGION (optional)
 
+This command will only install the agent on hosts that are supported by
+EC2InstanceConnect. The supported AMI types are Amazon Linux 2 and Ubuntu
+16.04 and later. There may also be a region restriction. You must be able
+to access the host's network.
```

Comment:
I think we can drop the `You must be able to access the host's network.` bit. I believe the correct wording is something like `The instance must be listening on an IP Address routable from the host running this command.` I think it may be better to drop this verbiage.

Reply:
Makes sense to me, removed it